specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -1535,6 +1934,137 @@
         }
       ]
     },
+    "ExtendedResourceProperties": {
+      "description": "The system generated resource properties associated with SQL databases and SQL containers.",
+      "type": "object",
+      "properties": {
+        "_rid": {
+          "type": "string",
+          "description": "A system generated property. A unique identifier."
+        },
+        "_ts": {
+          "description": "A system generated property that denotes the last updated timestamp of the resource."
+        },
+        "_self": {
+          "type": "string",
+          "description": "A system generated property. It is the unique addressable URI for the resource."
+        },
+        "_etag": {
+          "type": "string",
+          "description": "A system generated property representing the resource etag required for optimistic concurrency control."
+        }
+      }
+    },
+    "GenericResourceProperties": {
```

Comment:
nit; could we rename `GenericResourceProperties` -> `ProxyResource` to be consistent?